src/layout/section/seaSection/SeaSection.story.mdx

```diff
@@ -0,0 +1,97 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { Button } from 'button/index'
+import { ButtonGroup } from 'buttonGroup/index'
+import { Disclaimer } from 'disclaimer/index'
+import { ItemActionLabel } from 'itemActionLabel/index'
+import { Itinerary } from 'itinerary/index'
+import carpoolBlaBlaBusSmall from 'layout/section/heroSection/images-for-stories/carpool-bbb-small.svg'
+
+import carpoolBlaBlaBusLarge from './carpool-coral-bus-large.svg'
+import { SeaSection } from './index'
+
+export const ILLUSTRATIONS = {
+  'carpool-bla-bla-bus': {
+    large: carpoolBlaBlaBusLarge,
+    small: carpoolBlaBlaBusSmall,
+  },
+}
+
+export const Body = (
+<React.Fragment>
+  <Itinerary
+    places={[
+      {
+        time: '09:00',
+        isoDate: '2017-12-11T09:00',
+        stepAriaLabel: 'Pickup location',
+        mainLabel: 'Porte de Vincennes',
+        subLabel: 'Paris',
+        actionAriaLabel: '09:00 Paris Porte de Vincennes (New page with a map)',
+      },
+      {
+        time: '15:00',
+        isoDate: '2017-12-11T15:00',
+        stepAriaLabel: 'Dropoff location',
+        mainLabel: 'Gare Bordeaux Saint-Jean',
+        subLabel: 'Bordeaux',
+        connectionLabel: 'Change of vehicle',
+      },
+    ]}
+    small
+  />
+  <ButtonGroup>
+    <Button> Register with email </Button>
+    <Button> Register with Facebook </Button>
+  </ButtonGroup>
+  <Disclaimer as="p">
+    By clicking in register with email or Facebook you accept the terms of CGU and certify you
+    are older than 18.
+  </Disclaimer>
+  <ItemActionLabel labelTitle="Already a member?" action="Sign In" />
+</React.Fragment>
+)
+
+<Meta title="Sections/SeaSection" />
+
+# ** SEA Landing Pages **
+<Canvas>
+  <Story name="Default">
+    <SeaSection
+      leading="Et vous, qui allez-vous retrouver ?"
+      illustationUrl={{
+        small: ILLUSTRATIONS['carpool-bla-bla-bus'].small,
+        large: ILLUSTRATIONS['carpool-bla-bla-bus'].large,
+      }}
+    >
+      {Body}
+    </SeaSection>
+  </Story>
+</Canvas>
+
+## Specifications
+
+SEA Landing Pages responsive 2 Column Vertical Grid
+
+**Large**
+
+- `Media` tracker has 100%;
+- `Body` tracker has max-with `375px` and is vertical/horizontally centered
```

Comment:
Does that mean that if my screen is 500px for instance, I will only get content on 375px wide?

Reply:
It's probably a mistake in the design?  I don't think we had this behaviour on any page.
The width should grow with the screen when you are under 800px in small media mode.
Wdyt?

Reply:
@wakooka  Check it out [the layout](https://www.figma.com/file/zMeyZPxqu1SaosHAVuz6qs/Landing-Pages?node-id=150%3A648) it can better understand why the content has this `max-width`